benchmark/python/cast_storage.py

```diff
@@ -23,17 +23,17 @@ def measure_cost(repeat, f, *args, **kwargs):
 
 
 def run_cast_storage_synthetic():
-    def dns_to_csr(m, n, density, ctx, repeat):
+    def dense_to_sparse(m, n, density, ctx, repeat, storage_type):
```

Comment:
We changed all `storage_type` to 'stype` in front-end. Better to keep the term consistent.